src/contexts/contract/index.tsx

```diff
@@ -553,6 +553,7 @@ export const ContractProvider: React.FC<PropsWithChildren> = ({ children }) => {
       setPayload(formsData.current);
       setTxLoading(false);
     } else if (queueLength > 1) {
+      setTxLoading(false);
```

Comment:
that does not solve the underlying problem, which is the broadcast button enabled even with required fields empty